dist/tools/ethos/start_network.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+trap "cleanup" EXIT #SIGINT SIGTERM
+
+create_tap() {
+    ip tuntap add ${TAP} mode tap user ${USER}
+    sysctl -w net.ipv6.conf.tap0.forwarding=1
+    sysctl -w net.ipv6.conf.eth0.accept_ra=0
```

Comment:
That's nasty.
a) I want to receive RAs on my ethernet interface.
b) My ethernet interface is not called `eth0`.